libs/ledger/include/ledger/fees/storage_fee.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "ledger/fees/chargeable.hpp"
+
+namespace fetch {
+namespace ledger {
+
+class StorageFee : public Chargeable
```

Comment:
While I appreciate this is a fairly simple class, I would still split this into `.hpp` and `.cpp` files